components/x-content-block/src/ContentBlock.jsx

```diff
@@ -1,12 +1,58 @@
 import { h } from '@financial-times/x-engine';
 import styles from './ContentBlock.scss';
 
-const ContentBlock = (props) => (
-	<div className='x-content-block'>
-		<h1 className={styles.heading}>{props.title}</h1>
-		{props.isBreakingNews && <span>Breaking News!!!</span>}
-		<p dangerouslySetInnerHTML={{__html: props.content}} />
-	</div>
-);
+const ContentBlock = ({ mid, title, content, time, isBreakingNews, articleUrl, keytext, textrendered, authordisplayname }) => {
+	/**
+	 * Notes:
+	 *
+	 * * textrendered, authordisplayname and keytext properties are temporary for testing purposes.
+	 * 		Eventually, we would like to get rid of them.
+	 * * Currently, we don't use content property. We need to replace textrendered with content.
+	 * * At the moment article page does not pass articleUrl, therefore sharing links don't work.
+	 * * At the moment, title is rendered twice (once in this component and once in the post body).
```

Comment:
I've added the code you developed that removed the title from the post body so we can render it separately.